locations/jclouds/src/test/java/org/apache/brooklyn/location/jclouds/JcloudsProviderAndApiLoaderTest.java

```diff
@@ -68,6 +69,35 @@ public void testRegisteredProvider() throws Exception {
             ProviderRegistry.unregisterProvider(provider);
         }
     }
+
+    @Test
+    public void testRenamedRegisteredProvider() throws Exception {
+        String id = "my-example-provider";
+        String renamedId = "my-example-provider-renamed";
```

Comment:
Personal preference for using "oldId" and "newId". When I first read the variable names, It thought that "renamedId" was going to be what the provider had been renamed *to*, rather than what it was renamed *from*.